components/13-ironic/aio-values.yaml

```diff
@@ -47,12 +47,42 @@ conf:
       automated_clean: false
     dhcp:
       dhcp_provider: none
+    oslo_messaging_rabbit:
+      rabbit_ha_queues: true
   logging:
     logger_root:
       level: DEBUG
     logger_ironic:
       level: DEBUG
 
+endpoints:
+  oslo_messaging:
+    namespace: null
```

Comment:
This came from the genestack config file from keystone right? Cause that's actually wrong and been planning on deleting it.

Reply:
This came from understack keystone aio-values: https://github.com/rackerlabs/understack/blob/main/components/10-keystone/aio-values.yaml#L983-L1008

Reply:
I tried changing it all to just:
```
endpoints:
  oslo_messaging:
    hosts:
      default: rabbitmq-nodes
```
But it doesn't work...

Reply:
This one works:
```
endpoints:
  oslo_messaging:
    namespace: null
    statefulset:
      name: rabbitmq-server
    hosts:
      default: rabbitmq-nodes
```